<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiViewButtons</class>
 <widget class="QWidget" name="UiViewButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UiViewButtons</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_info">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>预约信息：</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>只看我的预约</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hboxlayout" stretch="0">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="40,60">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>152</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_1">
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>撤回预约</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_2">
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>新建预约</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_3">
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>退出登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
